recipes/greenlet/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "greenlet" %}
+{% set version = "0.4.12" %}
+{% set sha256 = "e4c99c6010a5d153d481fdaf63b8a0782825c0721506d880403a3b9b82ae347e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
Please drop `--single-version-externally-managed --record record.txt`. These are only needed if one uses `setuptools`, which it appears is not the case here.